kernel/rros/thread.rs

```diff
@@ -2259,10 +2223,10 @@ pub fn rros_find_sched_class(
     //         return ERR_PTR(-EINVAL);
     // }
 
-    // 按照librros写的，prio+ptid应为50，part取3
+    // According to what librros wrote, prio & ptid should be 50, and part should be 3.
```

Comment:
This is `prio + ptid`, not `prio & ptid`.

Reply:
OK